src/client/mod.rs

```diff
@@ -288,23 +309,44 @@ pub struct WrapData {
 pub struct AppRoleProperties {
     /// Require `secret_id` to be presented when logging in using this `AppRole`. Defaults to 'true'.
     pub bind_secret_id: bool,
+    /// Require `secret_id` to be presented when logging in using this `AppRole`. Defaults to 'true'.
+    pub local_secret_ids: bool,
     /// Comma-separated list of CIDR blocks; if set, specifies blocks of IP addresses which can
```

Comment:
If we're changing this type, the doc comment also needs updating

Reply:
Oh, thanks for catching that!

Reply:
This still is documenting a comma separated list, while expecting a Vec